packages/node-cli/src/utils/AuthenticationUtils.js

```diff
@@ -175,6 +179,25 @@ async function authenticateCi(params, sdkPath, projectFolder, executionEnvironme
 		.build();
 }
 
+async function selectAuthenticationCI(authId, sdkPath, projectFolder) {
+	//Validate authId exists into the CI file
+	const authIDActionResult = await getAuthIds(sdkPath);
+	if (!authIDActionResult.isSuccess()) {
+		throw authIDActionResult.errorMessages;
+	}
+	const authIDs = Object.keys(authIDActionResult.data);
+
+	if (authIDs.includes(authId)) {
+		setDefaultAuthentication(projectFolder, authId);
+		return AuthenticateActionResult.Builder.success()
+			.withMode(COMMANDS.AUTHENTICATE.MODES.REUSE)
+			.withAuthId(authId)
+			.build();
```

Comment:
Connected to `packages/node-cli/src/services/actionresult/AuthenticateActionResult.js` comment, I think we should add the accountInfo to the `AuthenticateActionResult` as I think it's available on `authIDs`.